src/client/dfs/dfs.c

```diff
@@ -1436,16 +1437,17 @@ dfs_cont_create_int(daos_handle_t poh, uuid_t *cuuid, bool uuid_is_set, uuid_t i
 		dattr.da_chunk_size = DFS_DEFAULT_CHUNK_SIZE;
 	}
 
+	pool = dc_hdl2pool(poh);
```

Comment:
just curious as why this change is needed here and in the other file?
to me dc_pool_get_redunc sounds cleaner then get the pool and then put the ref handle back.

Reply:
dc_pool_get_redunc() will issue an extra RPC to query pool, since 2.2 will break pool RPC proto anyway, i extend to pool connection return pool redunc property directly, so dc_pool_get_redunc() could be avoided.

Reply:
i see. would it be better to just then change dc_pool_get_redunc() or add a new function dc_pool_get_rf() that does what you are doing here instead? would save one from doing these steps to get pool handle, and release after to just get the rf.

Reply:
sure, make sense.